### PR TITLE
Feat/add babel scripts

### PR DESCRIPTION
**Pull request by shanesc**
This PR adds the Babel `build` script and the node `start` script.

### CODE REVIEW

**Review by r1903 — COMMENTED** [review]
I think we need to add a plugin to support arrow function

File: package.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
   "version": "1.0.0",
   "description": "Backend for Bristol Tech Fair.",
   "main": "index.js",
+  "scripts": {
+    "build": "babel src --out-dir dist",
```
Comment (r1903) [inline package.json:7]: Can we add script to restart application?

**Comment by shanesc** [comment]
> I think we need to add a plugin to support arrow function

I think this is handled with `@babel/preset-env` it loads support for ES6 syntax.

File: package.json (hunk 1 above)
Comment (shanesc) [reply package.json:7]: I found a way to reload on changes after running `yarn start` by using `nodemon` and `babel-node`.

``` json
"start": "nodemon --exec babel-node -r dotenv/config ./src/server.js"
```

I think it's a cleaner approach and more user friendly than using `restart`,  Iet me know what you think 😃

Comment (shanesc) [reply package.json:7]: I found a way to reload on changes by using `nodemon` and `babel-node`.

``` json
"start": "nodemon --exec babel-node -r dotenv/config ./src/server.js"
```

I think it's a cleaner approach and more user friendly than using `restart`,  Iet me know what you think 😃

Comment (r1903) [reply package.json:7]: That's great Shane. Its good idea to use nodemon in development env.

**Comment by r1903** [comment]
> > I think we need to add a plugin to support arrow function
> 
> I think this is handled with `@babel/preset-env` it loads support for ES6 syntax.

You are right.

**Review by r1903 — APPROVED** [review]

**Pull request merged**